documentation-site/examples/side-navigation/basic.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
```suggestion
```

Reply:
saw a couples instances of this

Reply:
yea, it should be in every single `.js` file since we want to enable flow typecheck?

Reply:
hah oh right